### PR TITLE
2048: styles update

### PR DESCRIPTION
**Pull request by Flmob**

### CODE REVIEW

File: games/2048/js/Tile.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ class Tile {
     if (this.value) {
       const textX = x + this.size / 2;
       const textY = y + this.size / 2;
-      this.ctx.font = `${this.size / 6}pt Arial`;
+      this.ctx.font = `${this.size / 6}pt Jura`;
```
Comment (a4b11s) [inline games/2048/js/Tile.js]: I think would be better move it to constans

Comment (Flmob) [reply games/2048/js/Tile.js]: It is a dinamic value. Size can be changed

Comment (a4b11s) [reply games/2048/js/Tile.js]: can Font family also be changed?

Comment (Flmob) [reply games/2048/js/Tile.js]: Got it!

**Review by a4b11s — APPROVED** [review]

**Pull request merged**